src/components/common/MovieDetail.tsx

```diff
@@ -0,0 +1,101 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { Formik, Form, Field, ErrorMessage } from "formik";
+import * as yup from "yup";
+import TextError from "./TextError";
+import axios from "axios";
+
+interface comentProps {
+	description: "";
+}
+
+export default function MovieDetail() {
+	let MovieDetail = {
```

Comment:
variable name should not be same like component and always start with smaller letter